appgate/resource_appgate_appliance.go

```diff
@@ -1033,6 +1043,11 @@ func resourceAppgateAppliance() *schema.Resource {
 											},
 										},
 									},
+									"labels_disabled": {
+										Type:     schema.TypeList,
+										Optional: true,
+										Elem:     schema.TypeString,
+									},
```

Comment:
Running this in 6.1 and 6.2 yields this error. We support 2 versions back so most likely missing a version checker in the code that touches this resource
```
=== NAME  TestAccAppliance62MetricsAggregator
    resource_appgate_appliance_test.go:4707: Step 1/2 error: Error running pre-apply refresh: exit status 1

        Error: metrics_aggregator.0.prometheus_exporter.0.labels_disabled: unknown element type (internal)

          with appgatesdp_appliance.appliancev62_metrics_aggregator,
          on terraform_plugin_test.tf line 6, in resource "appgatesdp_appliance" "appliancev62_metrics_aggregator":
           6: resource "appgatesdp_appliance" "appliancev62_metrics_aggregator" {
```